lib/AgrirouterClient/src/Protobuf/meson.build

```diff
@@ -0,0 +1,52 @@
+protoc_exe = find_program('protoc')
+
+pymodule = import('python3')
+python3 = pymodule.find_python()
+
+res = run_command(python3, '-c', 'import os; print(os.environ["PKG_CONFIG_SYSROOT_DIR"])')
```

Comment:
... but all you do with it is this. You can use `python3 = find_program('python3')` instead. It's much simpler, does the same thing, and isn't deprecated.

Reply:
I've changed it, thank you for the great review!